tests/framework/e2e/util.go

```diff
@@ -81,6 +81,25 @@ func SpawnWithExpectLines(ctx context.Context, args []string, envVars map[string
 	return lines, perr
 }
 
+// SpawnWithExpectResult is used to verify the output from short-running commands.
+func SpawnWithExpectResult(args []string, envVars map[string]string, f func([]string) error) error {
```

Comment:
Don't think we need a `f func([]string) error` argument, we can just return string from function.

Reply:
It is hard to verify the json in one query. If follow current pattern, we need to send three requests and verify it separately. It can't guarantee that cluster version won't change when we verify the storage version.

Reply:
Sorry for misundestanding. The suggestions looks good to me. Updated. PTAL.